pkg/endpointmanager/manager.go

```diff
@@ -183,6 +184,19 @@ func LookupIPv6(ipv6 string) *endpoint.Endpoint {
 	return ep
 }
 
+// LookupIP looks up endpoint by IP address
+func LookupIP(ip net.IP) (ep *endpoint.Endpoint) {
```

Comment:
Unused function? Usually we wouldn't backport this, though I guess it's harmless.

Reply:
Removed.